sklearn/inspection/tests/test_permutation_importance.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import numpy as np
+
+from sklearn.compose import ColumnTransformer
+from sklearn.datasets import load_boston, load_iris
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.inspection import permutation_importance
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.preprocessing import OneHotEncoder
+
+
+@pytest.mark.parametrize("to_pd", [True, False])
+def test_permutation_importance_correlated_feature_regression(to_pd):
+    rng = np.random.RandomState(42)
+
+    bunch = load_boston()
+    X, y = bunch.data, bunch.target
+    y_with_little_noise = (
+        y + rng.normal(scale=0.001, size=y.shape[0])).reshape(-1, 1)
+
+    # Adds feature correlated with target
+    if to_pd:
+        pd = pytest.importorskip("pandas")
+        X = pd.DataFrame(X, columns=bunch.feature_names)
+        X['correlated_feature'] = y_with_little_noise
+    else:
+        X = np.hstack([X, y_with_little_noise])
+
+    rf = RandomForestRegressor(n_estimators=10, random_state=rng)
+    rf.fit(X, y)
+    permute_scores = permutation_importance(rf, X, y, n_rounds=10,
+                                            random_state=rng,
+                                            scoring="neg_mean_absolute_error")
+
+    assert permute_scores.shape == (X.shape[1], 10)
+
+    permute_score_means = np.mean(permute_scores, axis=-1)
+
+    # correlated feature has the highest importance
+    assert np.all(permute_score_means[-1] > permute_score_means[:-1])
+
+
+def test_permutation_importance_correlated_feature_column_transframer():
```

Comment:
```suggestion
def test_permutation_importance_correlated_feature_column_transformer():
```

Reply:
I'm not sure of the benefit of this test compared to the other one? Seems redundant. If it's not, please add a comment to explain why